pkgs/development/web/playwright/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+, patchelf
+, google-chrome-dev
+, makeWrapper
+}:
+
+# Get rev from
+# https://github.com/microsoft/playwright/blob/v1.13.1/browsers.json
+let
+  chromium-rev = "901522";
+  firefox-rev = "1278";
+in
+
+stdenv.mkDerivation rec {
+  version = "1.13.1";
+  pname = "playwright";
+
+  src = fetchurl {
+    url = "https://playwright.azureedge.net/builds/chromium/${chromium-rev}/chromium-linux.zip";
```

Comment:
Is this a chromium binary? If so we should use the one from nixpkgs.

Reply:
It is. The main issue to create a playwright package is that playwright compiles the 3 browsers (chromium, firefox and webkit) with some patches. My idea here is to get the generated binaries and do what we are doing for google-chrome and firefox-bin. Ideally, I'd like to reuse what we are doing on those packages, that's why this derivation depends on google-chrome at build time